client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -27,55 +29,104 @@ import './style.scss';
+const connectedProps = ( translate, connectedMessage ) => ( {
 	iconPath: connectedIcon,
-	className: 'settings_connected',
+	className: 'settings__connected',
 	title: translate( 'Server status: Connected' ),
-	content: translate( 'One-click restores are enabled.' ),
+	content: connectedMessage,
 } );
 
-const disconnectedProps = ( translate ) => ( {
+const disconnectedProps = ( translate, disconnectedMessage ) => ( {
 	iconPath: disconnectedIcon,
-	className: 'settings_disconnected',
+	className: 'settings__disconnected',
 	title: translate( 'Server status: Not connected' ),
-	content: translate(
-		'Enter your server credentials to enable one-click restores for Backups. {{a}}Need help? Find your server credentials{{/a}}',
-		{
-			components: {
-				a: (
-					<ExternalLink
-						className="settings__link-external"
-						icon
-						href="https://jetpack.com/support/adding-credentials-to-jetpack/"
-					/>
-				),
-			},
-		}
-	),
+	content: disconnectedMessage,
 } );
 
-const ConnectionStatus = ( { isConnected } ) => {
-	const translate = useTranslate();
-	const cardProps = isConnected ? connectedProps( translate ) : disconnectedProps( translate );
+const getCardProps = ( isConnected, message, translate ) => {
+	if ( isConnected ) {
+		return connectedProps( translate, message );
+	}
+	return disconnectedProps( translate, message );
+};
 
-	return (
-		<Card compact={ true } className={ cardProps.className }>
-			<img className="settings__icon" src={ cardProps.iconPath } alt="" />
-			<div className="settings__details">
-				<div className="settings__details-head"> { cardProps.title } </div>
-				<div>{ cardProps.content }</div>
-			</div>
-		</Card>
-	);
+const ConnectionStatus = ( { cardProps } ) => (
+	<Card compact={ true } className={ cardProps.className }>
+		<img className="settings__icon" src={ cardProps.iconPath } alt="" />
+		<div className="settings__details">
+			<div className="settings__details-head"> { cardProps.title } </div>
+			<div>{ cardProps.content }</div>
+		</div>
+	</Card>
+);
+
+const getStatusMessage = ( isConnected, hasBackup, hasScan, translate ) => {
+	const HAS_BACKUP = 1;
+	const HAS_SCAN = 2;
+	const HAVE_BOTH = 3;
+
+	const helpLink = {
+		components: {
+			a: (
+				<ExternalLink
+					className="settings__link-external"
+					icon
+					href="https://jetpack.com/support/adding-credentials-to-jetpack/"
+				/>
+			),
+		},
+	};
+
+	const disconnectedMessages = {
+		[ HAS_BACKUP ]: translate(
+			'Enter your server credentials to enable one-click restores for backups. {{a}}Need help? Find your server credentials{{/a}}',
+			helpLink
+		),
+		[ HAS_SCAN ]: translate(
+			'Enter your server credentials to enable Jetpack to auto-fix threats. {{a}}Need help? Find your server credentials{{/a}}',
+			helpLink
+		),
+		[ HAVE_BOTH ]: translate(
+			'Enter your server credentials to enable one-click restores for backups and to auto-fix threats. {{a}}Need help? Find your server credentials{{/a}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Enter your server credentials to enable one-click restores for Backups. {{a}}Need help? Find your server credentials{{/a}}' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-cloud-copy-in-settings-server-connection)